src/openrct2/ride/Vehicle.cpp

```diff
@@ -8185,100 +8484,116 @@ static sint32 vehicle_update_track_motion_mini_golf(rct_vehicle *vehicle, sint32
     }
 
     // loc_6DC5B8
-    const rct_vehicle_info* moveInfo = vehicle_get_move_info(vehicle->var_CD, vehicle->track_type, 0);
+    // const rct_vehicle_info* moveInfo = vehicle_get_move_info(vehicle->var_CD, vehicle->track_type, 0);
```

Comment:
Why is this commented out?

Reply:
It wasn't used, but I wanted to leave it in because it comes from original code.

Reply:
Can you add a comment that says so?